test/unit/status/myDomainResolver.test.ts

```diff
@@ -142,60 +142,3 @@ describe('myDomainResolver', () => {
     expect(lookupAsyncSpy.callCount).to.be.equal(0);
   });
 });
-describe('cname resolver', () => {
```

Comment:
this isn't used by anything in salesforcecli or forcedotcom.

I know it's a breaking change to a public API.

Reply:
no public references https://github.com/search?q=myDomainResolver+cnames&type=code